models/DataObject/Concrete.php

```diff
@@ -733,12 +733,26 @@ public function __sleep(): array
 
     public function __wakeup(): void
     {
+        // parent::__wakeup() will call $this->setInDumpState(false) but we'll need the original value below
+        $wasInDumpState = $this->isInDumpState();
```

Comment:
@youwe-petervanderwal I guess there's no good reason for assigning a variable here, or is it?

Reply:
@brusch The reason to assign it to a variable is in the comment above ;)

The only alternative I have is moving the order around, so first set the object-aware fields and then call `parent::__wakeup()` but I didn't dare to do that because some of the `$this->$objectAwareField->setObject($this)` implementations had quite some logic in there on which I didn't knew in what way the relied on the connections that were set already in the `parent::__wakeup()`